server/src/main/java/org/candlepin/resource/HypervisorResource.java

```diff
@@ -110,6 +108,9 @@ public HypervisorResource(ConsumerResource consumerResource, ConsumerCurator con
         this.guestIdResource = guestIdResource;
 
         this.hypervisorType = consumerTypeCurator.lookupByLabel(ConsumerTypeEnum.HYPERVISOR.getLabel(), true);
+        if (this.hypervisorType == null) {
```

Comment:
Rebasing artifact; This block is unnecessary with the above boolean -- it'll do this creation if the type isn't found.